pace-util/tests/checkpointer/test_thresholds.py

```diff
@@ -0,0 +1,105 @@
+import numpy as np
+import pytest
+
+from pace.util.checkpointer import (
+    InsufficientTrialsError,
+    Threshold,
+    ThresholdCalibrationCheckpointer,
+)
+
+
+def test_thresholds_no_trials():
+    checkpointer = ThresholdCalibrationCheckpointer()
+    with pytest.raises(InsufficientTrialsError):
+        checkpointer.thresholds
+
+
+def test_thresholds_one_empty_trial():
+    checkpointer = ThresholdCalibrationCheckpointer()
+    with checkpointer.trial():
+        pass
+    with pytest.raises(InsufficientTrialsError):
+        checkpointer.thresholds
+
+
+def test_thresholds_two_empty_trials():
+    checkpointer = ThresholdCalibrationCheckpointer()
+    for _ in range(2):
```

Comment:
Shouldn't this raise InsufficientTrialsError also?

Reply:
Two trials is enough to generate thresholds.